backend/src/main/java/com/example/demo/CardService.java

```diff
@@ -0,0 +1,54 @@
+package com.example.demo;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class CardService {
+
+
+    private final CardRepository cardRepository;
+
+    public void createCard(Card card){
+        cardRepository.save(card);
+    }
+
+    public Collection<Card> getCards(){
+        return cardRepository.findAll()
+                .stream()
+                .sorted()
```

Comment:
I would let the database to the sorting. You can define a method like `findAllSortByName`, where name can be any field from your `Card` class. Perhaps the method name is a little different, but IntelliJ will help you.
That way it will probably be faster.

Reply:
Sounds good. Then I will change that

Reply:
Now, that the database does the sorting, you can remove the `.stream().sorted().toList()`

Reply:
Done